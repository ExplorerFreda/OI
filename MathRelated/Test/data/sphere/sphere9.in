10
16577.000000 17152.000000 10374.000000 3245.500000 3253.000000 1173.500000 14984.000000 6397.000000 7610.500000 1966.000000
12985.000000 12517.500000 5849.000000 8254.500000 18427.000000 1157.500000 8731.500000 14432.500000 18346.500000 6711.000000
2023.500000 3829.000000 5749.500000 2078.000000 12869.000000 10208.500000 17614.500000 972.000000 858.500000 10636.000000
14743.500000 12059.000000 14062.500000 1918.000000 4309.500000 13076.500000 16405.500000 12491.500000 15438.500000 16657.000000
5941.000000 15027.000000 10362.000000 1130.500000 19585.000000 3633.500000 15840.500000 7214.500000 13341.500000 15418.500000
15075.000000 13548.000000 13292.500000 18309.500000 5382.500000 4835.500000 6966.000000 2123.000000 8498.500000 2426.500000
16936.500000 15783.500000 16848.000000 15551.000000 3520.000000 19025.500000 1347.000000 799.000000 7853.500000 5052.000000
17032.000000 2034.000000 3772.500000 2086.500000 16890.500000 17483.500000 16219.000000 8837.000000 2416.000000 9701.500000
7709.500000 612.000000 17832.000000 14366.000000 12158.000000 5950.500000 10741.000000 9454.500000 13206.000000 13248.500000
19593.500000 6864.000000 15813.000000 17872.500000 19398.000000 3383.000000 101.500000 17915.000000 9319.500000 14277.000000
10218.500000 15403.000000 7755.000000 2560.500000 15362.500000 6818.000000 12254.500000 19481.000000 11969.500000 9861.500000
