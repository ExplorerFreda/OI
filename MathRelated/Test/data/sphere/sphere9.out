15007.824 6491.905 7385.310 -1984.447 8329.034 5311.464 -7089.265 2881.246 5553.146 16469.431
